### PR TITLE
global, last update patch, misc

### PR DESCRIPTION
**Pull request by ddarbs**
added global sheet creation, added global api pulls, changed last update to x minutes ago, organized comments, added menu buttons for rates and global along with an about button. 

Still relies on user having getData running on open.

#24 #25 #35

### CODE REVIEW

**Comment by ddarbs** [comment]
#40 fixed, fix detailed in #41 moved to here

**Comment by ddarbs** [comment]
Now relies on only one trigger, 'onOpen'.

File: scripteditor.js [hunk 1]
```diff
@@ -81,16 +93,13 @@ function getData() {
     ssRates.getRange('R1').setValue("Market Cap " + targetCurrency.toUpperCase());
   };
   
-  //   /\      /\      /\      /\       /\  
-  // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-
+/*     /\      /\      /\      /\       /\
+     ============================================== */
```
Comment (LesterCovax) [inline scripteditor.js:97]: extra comment tags

File: scripteditor.js [hunk 2]
```diff
@@ -114,169 +123,184 @@ function getData() {
...
 
-  
-  //
-  // ===== Wallet Sheet Creator ========================================
-  // Uncomment the lines of code below
-  // It will create the Wallets sheet for you
-  // If using the Wallets sheet ALWAYS leave it uncommented
-  // ===================================================================
+/*     /\      /\      /\      /\       /\
+     ============================================== */
```
Comment (LesterCovax) [inline scripteditor.js:138]: extra comment tags

File: scripteditor.js [hunk 3]
```diff
@@ -306,21 +329,28 @@ function getLtcBalance(ltcAddress) {
...
-// !!! DEPRECATED !!!
-// USE AT YOUR OWN RISK
-// XXXXXXXXXXXXXXXXXXXXXXX
+function about() {
+  SpreadsheetApp.getUi()
+     .alert('Visit https://github.com/saitei/crypto-sheets to get the latest dev build, report issues, or request new features!');
+}
+
+/*     /\      /\      /\      /\       /\
+     ============================================== */
```
Comment (LesterCovax) [inline scripteditor.js:348]: extra comment tags

**Review by LesterCovax — APPROVED** [review]
Looks great, works great, nice job with all the features added!

**Pull request merged**